runtime/compiler/z/runtime/s390_asdef.inc

```diff
@@ -61,6 +61,7 @@ define([LNR_GPR],lngr)
 define([LTR_GPR],ltgr)
 define([MSGR_GPR],msgr)
 define([N_GPR],ng)
+define([NR_GPR],ngr)
 define([NIHH_GPR],nihh)
```

Comment:
Can we alphabetize the newly added line? Similarly below.

Reply:
Fixed.